packages/frontend/src/components/TableTree.tsx

```diff
@@ -501,7 +505,25 @@ const TableTree: FC<TableTreeProps> = ({
                 <strong>Metrics</strong>
             </span>
         ),
-        secondaryLabel: null,
+        secondaryLabel: hasNoMetrics ? (
+            <DocumentationHelpButton
+                url={
+                    'https://docs.lightdash.com/get-started/setup-lightdash/add-metrics/#2-add-a-metric-to-your-project'
+                }
+                tooltipProps={{
+                    content: (
+                        <TooltipContent>
+                            <b>View docs</b> - Add a metric to your project
+                        </TooltipContent>
+                    ),
+                }}
+                iconProps={{
+                    style: {
```

Comment:
can this inline style be `wrapped`? (not sure if possible in this iconProps)

Reply:
By wrapped you mean using styled components ? I don't think so 🤔